crates/agglayer-types/src/lib.rs

```diff
@@ -423,7 +423,8 @@ impl Certificate {
             .first()
             .map(|imported_bridge_exit| imported_bridge_exit.l1_info_root())
         else {
-            return Ok(None);
+            // In case of no imported bridge exists, return the provided l1_info_root
+            return Ok(self.l1_info_root);
```

Comment:
Not urgent/blocking but I'd also compare it with the one inferred from the imported bridge exits a bit below, for the case where we do have imported bridge exits